opacus/privacy_engine.py

```diff
@@ -258,21 +260,23 @@ def make_private(
         loss_reduction: str = "mean",
         poisson_sampling: bool = True,
         clipping: str = "flat",
+        noise_generator=None,
     ) -> Tuple[GradSampleModule, DPOptimizer, DataLoader]:
         """
         Add privacy-related responsibilites to the main PyTorch training objects:
-        model, optimizer and the data loader.
+        model, optimizer, and the data loader.
 
-        All of the returned objects act just like treir non-private counterparts
+        All of the returned objects act just like their non-private counterparts
         passed as arguments, but with added DP tasks.
 
         Model is wrapped to also compute per sample gradients.
         Optimizer is now responsible for gradient clipping and adding noise to the
-            gradients
+            gradients.
         DataLoader is updated to perform Poisson sampling.
 
-        Note, that using any other models, optimizers or data sources during training
-        will invalidate stated privacy guarantees.
+        Notes:
```

Comment:
nit: new line is required before this line (you can just merge this PR and worry about this later)